app/lib/shared/integrity.dart

```diff
@@ -93,7 +98,13 @@ class IntegrityChecker {
       yield* _checkAuditLogs();
       yield* _checkModerationCases();
       yield* _reportPubspecVersionIssues();
-      // TODO: report unmapped properties
+
+      if (_unmappedFields.isNotEmpty) {
+        for (final field in _unmappedFields) {
+          if (_knownUnmappedFields.contains(field)) continue;
+        yield 'Unmapped field found: $field.';
```

Comment:
Can we report more? Like give some entity with this field, and the entitty type

Reply:
We are reporting `EntityType.fieldName` here. I'm not sure if we need to have a full entity identifier, as these fields are typically unused, but maybe we could report one id for each of the reported entries? That way we can store it in a Map and only keep the first one?

Reply:
Implemented this, PTAL.